rts/lra-examples/cdi-participant/pom.xml

```diff
@@ -95,22 +95,28 @@
         <dependency>
             <groupId>org.jboss.narayana.rts</groupId>
             <artifactId>narayana-lra</artifactId>
+            <version>${project.version}</version></dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-resteasy-jackson</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-rest-client-jackson</artifactId>
+        </dependency>
+        
+        <dependency>
+            <groupId>org.jboss.narayana.rts</groupId>            
+            <artifactId>narayana-lra</artifactId>
             <version>${project.version}</version>
         </dependency>
-
-        <!-- MP LRA CDI support defines annotations for controlling the lifecycle
-             of LRAs and for controlling participation in an LRA
-             -->
         <dependency>
-            <groupId>org.eclipse.microprofile.lra</groupId>
-            <artifactId>microprofile-lra-api</artifactId>
-            <version>${version.microprofile.lra}</version>
+          <groupId>org.jboss.narayana.rts</groupId>
+          <artifactId>lra-service-base</artifactId>
```

Comment:
Please can you clarify why the MP LRA API is removed and the lra-service-base is added?

Reply:
the MP LRA API doesn't seem to be necessary, OTOH lra-service-base dependency is needed ( see [#395 (comment)](https://github.com/jbosstm/quickstart/pull/395#issuecomment-1190305567))